scipy/integrate/_quad_vec.py

```diff
@@ -102,7 +102,8 @@ def __repr__(self):
 
 
 def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6, limit=10000,
-             workers=1, points=None, quadrature=None, full_output=False):
+             workers=1, points=None, quadrature=None, full_output=False,
+             args=()):
```

Comment:
```suggestion
             *, args=()):
```

Reply:
I will make that keyword only, makes sense.

Reply:
FYI, you could have just use the suggestion directly instead of doing the work again on your side... As for the contribution goes, the commit would have automatically been marked as a co-authored commit. Which IMO is a good thing as we are both taking some time here.